doc/Doxyfile.template

```diff
@@ -26,13 +26,13 @@ DOXYFILE_ENCODING      = UTF-8
 # identify the project. Note that if you do not use Doxywizard you need
 # to put quotes around the project name if it contains spaces.
 
-PROJECT_NAME           = "___DOC_NAME___"
+PROJECT_NAME           = "Wayland InVehicle Infotainment Extension"
 
 # The PROJECT_NUMBER tag can be used to enter a project or revision number.
 # This could be handy for archiving the generated documentation or
 # if some version control system is used.
 
-PROJECT_NUMBER         = ___DOC_VERSION___
+PROJECT_NUMBER         = 0.0.2
```

Comment:
it should be 2.2.0, in general it would make sence to rework the project number assingment and make it centralized at one place but i would prefer to do it in a separate commit...
so here just change to 2.2.0